sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -482,6 +507,24 @@ support using this graph extension.
 
 |===
 
+Command-graph update support may also not be available across all devices. The
+following extension to the `aspect` enum class can be used to query for graph
+update support:
+
+[source]
+----
+namespace sycl {
+enum class aspect {
+  ...
+  sycl_ext_oneapi_graph_update
+}
+}
+----
+
+If a SYCL device has the `sycl_ext_oneapi_graph_update` aspact, then it supports
+the functionality described in the <<executable-graph-update, Executable Graph
+Update>> section.
+
```

Comment:
I thought we discussed these aspects the other day and agreed to do the following:

* Add an aspect `ext_oneapi_graph` documented like:

  > This aspect indicates that the device supports all of the APIs in this extension,

* Add a temporary aspect `ext_oneapi_limited_graph` documented like:

  > This aspect indicates that the device supports all of the APIs in this extension except for the APIs described in the Executable Graph Update section.  This is a temporary aspect that we intend to remove once devices with full graph support are more prevalent.

Reply:
Apologies I was not aware of those specific points of discussion. The `ext_oneapi_graph` aspect is being added as part of https://github.com/intel/llvm/pull/12812 which will almost certainly merge before this PR, so I've left it for now and will update this PR once that change is in.

For now I have updated the name and description of the update related aspect as previously discussed.

Reply:
When #12812 is merged, can you update this PR so that both aspects are described in a consistent way?  For example, #12812 adds a table for the new aspect, but this PR adds a code synopsis instead.  It would be better to describe both aspects in the same table or show both in the same code synopsis.

Reply:
The aspects should be consistent now.